### PR TITLE
feat: init repo with pgvecto.rs support

### PR DESCRIPTION
**Pull request by kemingy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
 # Vector DB Benchmark
+
+Supported databases/extensions:
+
+- [x] [`pgvecto.rs`](https://github.com/tensorchord/pgvecto.rs)
+- [ ] [`pgvector`]()
```
Comment (gaocegege) [inline README.md]: Could we add the link here?

Comment (gaocegege) [reply README.md]: Don't we support other specialized vdbs?

Comment (kemingy) [reply README.md]: The plan is to compare with `pgvector` and `qdrant` which are open-sourced. Will add the impl later. This PR is mainly to check whether the user interface is okay (see the usage in README)

Comment (gaocegege) [reply README.md]: LGTM

**Comment by gaocegege** [comment]
LGTM

**Pull request merged**